src/voodoo/utils.ts

```diff
@@ -0,0 +1,20 @@
+import { utils } from 'ethers'
+import { buildContentTopic } from '../utils'
```

Comment:
Just to further avoid conflicts, and to make the topic restriction rules easier to work with, what do we think about bumping the version in the root of the proto to `xmtp/1/`

Reply:
Sounds good to me, currently not using any protos which is probably questionable

Reply:
Oh, yeah I'll make the change to the version number!